base/bootstrap/include/motis/bootstrap/import_status.h

```diff
@@ -24,7 +24,11 @@ struct import_status : public module::progress_listener {
   bool update(motis::module::msg_ptr const&);
   void print();
 
-  void update_progress(std::string const& name, unsigned progress) override;
+  void set_progress_bounds(std::string const& name, double output_low = 0,
+                           double output_high = 100,
```

Comment:
```suggestion
                           double output_high = 100.,
```

Brauchen wir wirklich 'double' Precision? Für die Prozentzahl am Ende reicht 'float'.

Reply:
brauchen wir uns wirklich gedanken zu machen?
ist es kriegsentscheident etwas anderes als den default zu nehmen?

Reply:
wusste nicht, dass double default ist. habe bisher immer float verwendet

Reply:
bei `int` und `unsigned` verwenden wir ja auch nicht `ssize_t`/`size_t` bzw. `int64_t`/`uint64_t` - weil die meisten berechnungen das nicht brauchen.
genauso brauchen die meisten berechnungen kein `double`. warum sollten wir da nicht auch `float` als standard verwenden und nur auf `double` wechseln, wenn es erforderlich ist.
c++ standard strategie: der default ist schnell

Reply:
k